test/handler.test.js

```diff
@@ -195,6 +195,24 @@ describe('strong-error-handler', function() {
       });
     });
 
+    it('contains thrown message when debug=true', function(done) {
+      var error = new Error('Thrown message');
+      givenErrorHandlerForError(error, {debug: true});
+      requestJson().end(function(err, res) {
+        expect(res.body.error.message).eql('Thrown message');
+        done();
+      });
+    });
+
+    it('returns default message when debug=false', function(done) {
```

Comment:
I think this test is redundant, as this part is already covered by the test "contains only safe info when status=5xx" below.